src/app/containers/CpsAssetPageMain/index.test.jsx

```diff
@@ -1,31 +1,43 @@
 import React from 'react';
 import { StaticRouter } from 'react-router-dom';
-import { matchSnapshotAsync } from '@bbc/psammead-test-helpers';
+import {
+  matchSnapshotAsync,
+  suppressPropWarnings,
+} from '@bbc/psammead-test-helpers';
 import { ServiceContextProvider } from '#contexts/ServiceContext';
 import { RequestContextProvider } from '#contexts/RequestContext';
 import CpsAssetPageMain from '.';
 import preprocessor from '#lib/utilities/preprocessor';
-import pidginPageData from '#data/pidgin/cpsAssets/tori-49450859';
+import pidginPageData from '#data/igbo/cpsAssets/afirika-23252735';
```

Comment:
Debating if we should have a snapshot for STY (`igbo/afirika-23252735`) and a snapshot for MAP (`pidgin/tori-49450859`) 🤔

Reply:
Done :)